src/pages/experiments/[experimentId]/data-processing/data-integration/components/SeuratV4Options.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Form, InputNumber, Select } from 'antd';
+
+const { Option } = Select;
+
+const MethodOptions = (props) => {
```

Comment:
New method-specific component to facilitate changes in options when different component method is chosen